regulations/static/regulations/css/less/module/comment.less

```diff
@@ -684,31 +757,28 @@ a.comment-index-review {
     }
   }
 
+  .comment-controls  {
+    display: none;
+  }
```

Comment:
Typical browser width doesn't go below 500 pixels, but to be safe I just put a max device width to hide the comment controls. So now desktops that scrunch to a very narrow width should be still able to see the attachment controls but tablets/phones shouldn't be able to now.